transformations/whitespace_perturbation/transformation.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def whitespace(text, remove_prob=0.1, add_prob=0.05, seed=0):
+    random.seed(seed)
```

Comment:
`random.seed()` should be called once before each call of `random.random()` for it to work as expected.

Reply:
It might also be better to generate a range of random integers and use them as seeds for the `random()` calls below so you can introduce some variety into the randomization while still making the randomization reproducible.